eralchemy/cst.py

```diff
@@ -1,7 +1,8 @@
 """All the constants used in the module."""
 
 TABLE = (
-    '"{}" [label=<<FONT FACE="Helvetica"><TABLE BORDER="0" CELLBORDER="1"'
+    #'"{}" [label=<<FONT FACE="Arial;Helvetica Neue;Helvetica;"><TABLE BORDER="0" CELLBORDER="1"'
+    '"{}" [label=<<FONT FACE="Helvetica;"><TABLE BORDER="0" CELLBORDER="1"'
```

Comment:
Why is this change required? I don't think that this has to do with PlantUML, does it?